solidity/migrations/2_deploy_contracts.js

```diff
@@ -27,7 +27,6 @@ module.exports = async function(deployer, network) {
   // Set the stake initialization period to 1 block for local development and testnet.
   if (network === 'local' || network === 'ropsten' || network === 'keep_dev') {
     initializationPeriod = 1;
-    undelegationPeriod = 60;
```

Comment:
Don't we want to keep this value lower for the test environments than for production? Having that helped us to find the bug we're fixing in this PR.

Reply:
We can make it lower but only if we merge changes from https://github.com/keep-network/keep-core/issues/1545 first. I'd propose to merge this PR as-is and after on-chain parameters are updated, consider changing this value for testnet.